src/commands/settings/settings.ts

```diff
@@ -6,15 +6,14 @@
  */
 
 import * as vscode from 'vscode';
-import { getExtensionName } from '../../utils/workspaceUtils';
 
 const SETTING_KEY_SUPPRESS_ALL = 'suppressAll';
 const SETTING_KEY_SUPPRESS_BY_RULE_ID = 'suppressByRuleId';
 
 export function getUpdateDiagnosticsSettingCommand(
     context: vscode.ExtensionContext
 ): string {
-    return `${getExtensionName(context)}.updateDiagnosticsSetting`;
```

Comment:
`getExtensionName()` was only used here so no need to have a method defined in `workspaceUtils.ts`.